.github/workflows/docs.yml

```diff
@@ -0,0 +1,30 @@
+# Publish documentation
+
+name: Publish documentation
+
+on:
+  push:
+    branches:
+      - 'main'
+      #- 'docs/**'
+      - 'feature/**'
```

Comment:
Why not the other way round? Don't we want to deploy if the branch is called `docs/**` ?

Reply:
Nope, because we want to deploy to GH Pages as soon as the pr gets merged into the `main` branch. If we deploy to Pages for every push into a `docs` branch, the documentation would be inconsistent. Maybe for the future we should setup a dev version of the documentation, which reflects the latest changes.
But I agree with you to uncomment the `feature` branches. This was only for testing.

Reply:
ah true, this is for deployment, `docs/**` wouldn't make sense then...